honeycombio/type_helpers.go

```diff
@@ -193,11 +194,19 @@ func expandNotificationRecipients(s []interface{}) []honeycombio.NotificationRec
 			// notification details have been provided
 			details := v[0].(map[string]interface{})
 			if s, ok := details["pagerduty_severity"]; ok {
-				recipients[i].Details = &honeycombio.NotificationRecipientDetails{
+				rcpt.Details = &honeycombio.NotificationRecipientDetails{
 					PDSeverity: honeycombio.PagerDutySeverity(s.(string)),
 				}
 			}
+		} else {
+			// set default notification details
+			if rcpt.Type == honeycombio.RecipientTypePagerDuty {
+				rcpt.Details = &honeycombio.NotificationRecipientDetails{
+					PDSeverity: honeycombio.PDSeverityCRITICAL,
```

Comment:
Where is this used such that we need to hard-code the default here. It'd be nice if we could rely on the API to provide this and not have to duplicate that logic.

Reply:
Nice catch! This is basically unused because we're already reading back from the API when the function is called